datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py

```diff
@@ -770,6 +791,45 @@ def _submit_gauges_from_histogram(self, metric_name, metric, scraper_config, hos
                         hostname=custom_hostname,
                     )
 
+        # in addition to sending distribution buckets, we might also want to send the cumlative bucket counters
+        # here we know `send_histogram_counters_with_distributions` var is True, and therefore all the other vars
+        # so we can reduce all the checks we need to do
+        if cumulative_metric is not None:
```

Comment:
I don't love this approach since it feels like a lot of code was duplicated, but its slightly different since we can assume a lot about the config vars once we're here.